website/docs/migrations/migrate-from-ckb-sdk-core.md

```diff
@@ -0,0 +1,388 @@
+# Migrate From ckb-sdk-core to Lumos
+
+## Summary
+To avoid splitting the development ecosystem, ckb-sdk-js will be merged with lumos, please check the migration list for migration solution. 
+
+If you are using a method that is not on this list, please refer to the lumos documentation for more help
```

Comment:
If you are using any function that is not on this list, please refer to the `lumos` documentation.